django/core/checks/security/base.py

```diff
@@ -194,6 +201,19 @@ def check_secret_key(app_configs, **kwargs):
 
 
 @register(Tags.security, deploy=True)
+def check_secret_key_valid_unicode(app_configs, **kwargs):
+    valid_unicode = True
+    try:
+        # To maintain backwards compatibility, instead of checking that
+        # SECRET_KEY is a valid Unicode string, simply check if it goes through
+        # `force_text` without throwing an exception.
+        force_text(settings.SECRET_KEY)
```

Comment:
Force text means, that it does not need to be a unicode string. I could be a python 2 string or utf-8 bytes.
I guess you should ether change the documentation, or even better be more strict about this warning.